featurebyte/service/batch_request_table.py

```diff
@@ -73,7 +73,7 @@ async def get_batch_request_table_task_payload(
             await self.context_service.get_document(document_id=data.context_id)
 
         return BatchRequestTableTaskPayload(
-            **data.dict(),
+            **data.dict(by_alias=True),
```

Comment:
Curious why is `by_alias` required here and what would be the issue if that is not set? I don't fully follow how it works end to end.

Reply:
Basically,
```
feat.dict()                           =>  {...., "internal_graph": [...], ...}
feat.dict(by_alias=True)   =>  {...., "graph": [...], ...}
```

Reply:
When the `alias` is used, the constructor parameters expect `alias` but not the attribute value. Outside of the model (like dictionary, json, constructor parameters), we should use alias key. When referring to the attribute, we should use not attribute value. (`root_validator(pre=False)` is a special case here as non-alias key is used inside the `values`).

Reply:
> When the alias is used, the constructor parameters expect alias but not the attribute value.

I see, I wasn't aware of this. Thanks for the explanation!